src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.ts

```diff
@@ -846,4 +856,24 @@ export class TicketInfoComponent implements OnInit {
         if (error.status == 409) this.conflict = true;
       });
   }
+   
+  /**
+   * Returns the full section text from a ViolationTicketCount, ie 44(3)(a)(iii)
+   * @param vtc 
+   * @returns 
+   */
+  private getSectionText(vtc: ViolationTicketCount) : string {
+    let sectionText = "";
+    if (vtc.section) sectionText = vtc.section;
```

Comment:
section maybe `null` ?
If yes, `let sectionText  = vtc.section ?? "";`

Reply:
Nice. Learning all kinds of angular syntax. Thanks.